Description: use separately packaged JavaScript libraries
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: not-needed
Last-Update: 2016-08-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/index.html
+++ b/test/index.html
@@ -6,10 +6,10 @@
   <link rel="stylesheet" href="vendor/qunit.css" type="text/css" media="screen">
 </head>
 <body>
-  <script src="vendor/qunit.js"></script>
-  <script src="vendor/jquery.js"></script>
-  <script src="vendor/json2.js"></script>
-  <script src="vendor/underscore.js"></script>
+  <script src="/usr/share/javascript/qunit/qunit.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.js"></script>
+  <script src="/usr/share/javascript/json/json2.js"></script>
+  <script src="/usr/share/javascript/underscore/underscore.js"></script>
   <script src="../backbone.js"></script>
   <script src="setup/dom-setup.js"></script>
   <script src="setup/environment.js"></script>
--- a/examples/todos/index.html
+++ b/examples/todos/index.html
@@ -40,10 +40,10 @@
     <br />Rewritten by: <a href="https://github.com/tastejs/todomvc">TodoMVC</a>.
   </div>
 
-  <script src="../../test/vendor/json2.js"></script>
-  <script src="../../test/vendor/jquery.js"></script>
-  <script src="../../test/vendor/underscore.js"></script>
-  <script src="../../backbone.js"></script>
+  <script src="/usr/share/javascript/json/json2.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.js"></script>
+  <script src="/usr/share/javascript/underscore/underscore.js"></script>
+  <script src="/usr/share/javascript/backbone/backbone.js"></script>
   <script src="../backbone.localStorage.js"></script>
   <script src="todos.js"></script>
 
--- a/index.html
+++ b/index.html
@@ -5235,11 +5235,11 @@
 
   </div>
 
-  <script src="test/vendor/underscore.js"></script>
-  <script src="test/vendor/jquery.js"></script>
-  <script src="docs/js/jquery.lazyload.js"></script>
-  <script src="test/vendor/json2.js"></script>
-  <script src="backbone.js"></script>
+  <script src="/usr/share/javascript/underscore/underscore.js"></script>
+  <script src="/usr/share/javascript/jquery/jquery.js"></script>
+  <script src="/usr/share/javascript/jquery-lazyload/jquery.lazyload.js"></script>
+  <script src="/usr/share/javascript/json/json2.js"></script>
+  <script src="/usr/share/javascript/backbone/backbone.js"></script>
   <script src="docs/search.js"></script>
 
   <script>
--- a/karma.conf.js
+++ b/karma.conf.js
@@ -9,9 +9,9 @@
 
     // list of files / patterns to load in the browser
     files: [
-        'test/vendor/jquery.js',
-        'test/vendor/json2.js',
-        'test/vendor/underscore.js',
+        '/usr/share/javascript/jquery/jquery.js',
+        '/usr/share/javascript/json/json2.js',
+        '/usr/share/javascript/underscore/underscore.js',
         'backbone.js',
         'test/setup/*.js',
         'test/*.js'
